src/appleseed.shaders/src/maya/as_maya_anisotropic.osl

```diff
@@ -269,7 +269,7 @@ shader as_maya_anisotropic
                 );
         }
 
-        if (transparency)
+        if (max(transparency) > 0.0)
```

Comment:
Yes, in the case we want to optimize transparency is constant, equal to 0. If OSL can remove the test and the closure assignment, the closure does not appear in the used closures list and 
we know we can skip shader exec. for shadows and transparency.

I don't mind if (color != 0) if OSL can constant fold it correctly. That's something we need to test.